CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ All notable changes to this project will be documented in this file.
   `criticalup run $WHATEVER` and have the respective tool see the appropriate CriticalUp-managed tools
   within the `$PATH` (or equivalent). A `--strict` flag was added to make it possible to ensure only
   tools within the installation are run.
+- Argument names for `criticalup run` and `criticalup which` commands are now consistent, both using `<COMMAND>`.
```

Comment:
Not sure this needs to be in the changelog, it's not noticable unless customers are grepping our `--help` args.